webcompat/templates/home-page/form.html

```diff
@@ -85,7 +85,7 @@ <h1 class="headline-1">Report Site Issue</h1>
         {{ subradio.label(class_='form-label label-radio') }}
       {% endfor %}
     </div>
-    <div class="form-upload js-image-upload js-Form-group loader js-image-loader">
+    <div class="form-upload js-image-upload js-Form-group">
```

Comment:
I think this is wrong too, https://github.com/webcompat/webcompat.com/commit/1444cedd5c630c3c191348068645c29fb5a129e2

Reply:
(i'm fixing this)